### DISCUSSION TITLE
is there a 'mono-descargar' library in Tampermonkey?

### DISCUSSION THREAD

**Discussion by yuanlin-yu** [Tampermonkey/tampermonkey#2225] [Q&A]
I had downloaded a user script lately, and when I am looking into the code from the script, it contains  `var mono = window['mono-descargar'];`. What I want to ask is that if the whole 'mono-descargar' thing is a library or something provided by Tampermonkey? 
_Because when I use this  `var mono = window['mono-descargar'];` in a new blank chrome extension script, it turn out to be undefined. :D._

**Comment by derjanb** [comment]
> is there a 'mono-descargar' library in Tampermonkey?

No. Seems  to be something site specific.
